packages/components/psammead-brand/src/index.jsx

```diff
@@ -71,6 +71,24 @@ const BrandSvg = styled.svg`
   /* stylelint-enable */
 `;
 
+const LocalisedBrandName = ({ brandName }) => {
+  const defaultBrandName = 'BBC News';
+
+  if (brandName !== defaultBrandName) {
+    return (
+      <Fragment>
+        <span lang="en-GB">{defaultBrandName}</span>
+        {// get substring after defaultBrandName
+        brandName.substring(
```

Comment:
This PR can benefit from the work done in https://github.com/bbc/simorgh/pull/1869 to start using the config
```
{
  product: 'BBC News',
  serviceLocalizedName: 'Ìgbò',
}
```

Sadly we cant rely on `BBC News` as eventually this component may need to support other services like `BBC Sport` etc. Could we pass in these 2 values into the component? Feel free to rename them, and ofcourse `serviceLocalizedName` will need to be optional for the core `BBC News` site :)

Reply:
Makes sense, this calls for changes on simorgh as well, I'll have both PRs ready then we can use [that ](https://github.com/bbc/simorgh/pull/1869) implementation as soon as it is merged.